.github/workflows/ci.yaml

```diff
@@ -32,6 +32,27 @@ jobs:
         with:
           config: ./typos.toml
 
+  docker-build:
+    runs-on: ubuntu-18.04
```

Comment:
Maybe we need to update the os version? it's out-of-date.

The ubuntu-18.04 environment is deprecated, consider switching to ubuntu-20.04(ubuntu-latest), or ubuntu-22.04 instead. For more details see https://github.com/actions/virtual-environments/issues/6002